data-catalog-backend-app/src/main/resources/application-test.properties

```diff
@@ -2,19 +2,19 @@ github.keyPath=../../travis/datajegerne-private-key.pk8
 
 # Database
 spring.datasource.driver-class-name=org.h2.Driver
-spring.datasource.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1
+spring.datasource.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1;MODE=PostgreSQL;INIT=CREATE SCHEMA IF NOT EXISTS BACKEND_SCHEMA
```

Comment:
Gjelder denne endringen kun for "test" eller bør vi legge inn den i "i-test" og "local" også?

Trenger vi "INIT" når vi har 
"CREATE SCHEMA IF NOT EXISTS BACKEND_SCHEMA;" i filen V1.0__initialize_schema_and_tables.sql?

Reply:
Mener den kun gjelder test, det er bare da vi bruker h2 - databasen. H2 støtter ikke function based index ( upper(name) ), så da måtte jeg konfigurere om test-profilen til ikke å bruke flyway. Dermed må skjemaet opprettes "manuelt"